gpt_index/playground/base.py

```diff
@@ -144,3 +145,80 @@ def compare(
             return pd.DataFrame(result)
         else:
             return result
+
+    def compare_with_optimizer(
+        self, query_text: str, to_pandas: Optional[bool] = True
+    ) -> Union[pd.DataFrame, List[Dict[str, Any]]]:
+        """Compare index outputs on an input query.
+
+        Args:
+            query_text (str): Query to run all indices on.
+            to_pandas (Optional[bool]): Return results in a pandas dataframe.
+                True by default.
+
+        Returns:
+            The output of each index along with other data, such as the time it took to
+            compute. Results are stored in a Pandas Dataframe or a list of Dicts.
+        """
+        print(f"\033[1mQuery:\033[0m\n{query_text}\n")
```

Comment:
i'd remove print statements, replace with logging (we might add back print later with a `verbose` flag)

Reply:
Hmm this was part of the original playground script. Should I change their whole file?

Reply:
oh actually nvm disregard this comment re: playground